java-squid/src/main/java/org/sonar/java/JavaFilesCache.java

```diff
@@ -184,7 +184,7 @@ private void handleSuppressWarning(List<AnnotationTree> annotationTrees, int end
   }
 
   private static boolean isSuppressWarningsAnnotation(AnnotationTree annotationTree) {
-    return annotationTree.annotationType().symbolType().is("java.lang.SuppressWarnings");
+    return annotationTree.annotationType().symbolType().is("java.lang.SuppressWarnings") && annotationTree.arguments().size() > 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1155)